modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/IgniteBinaryMetadataUpdateNodeRestartTest.java

```diff
@@ -47,6 +48,7 @@
 /**
  *
  */
+@Ignore("IGNITE-4768, IGNITE-9214")
```

Comment:
Here and elsewhere: please choose a single most relevant ticket and put the complete JIRA URL in @Ignore message. TC/MTCGA wll recognize and link it.

Reply:
@alamar,
> please choose a single most relevant ticket 
disagree here, more hints = more chances to have this done in future
> put the complete JIRA URL
+1 here

Reply:
Do we support more than one JIRA URL in our TC/MTCGA tooling?

Reply:
I found that there are cases with comma separated list of links. For example `PageEvictionTouchOrderTest.testTouchOrderWithFairFifoEvictionMvccTxReplicated`. But MTCGA shows only first ticket. Also I try find some tests with double @Ignore annotation, but there are no such cases.

Looks like we should continue the practice, use csv for such things. Also MTCGA should display them correctly, I can create a ticket for that.

WDYT?

Reply:
Added csv jira urls.

Reply:
Create a jira ticket for the display: https://issues.apache.org/jira/browse/IGNITE-13762